### DISCUSSION TITLE
How to Add an 'Add Comments' Link Under Each Blog Post on the Main Blog Page?

### DISCUSSION THREAD

**Discussion by ruanqizhen** [facebook/docusaurus#10802] [Q&A]
On the main blog page, under each post, there is currently a "tags" section. Is it possible to add an additional link, such as "Add Comments" or something similar, alongside the tags?  Or if there is a way to customize the texts for "Read More"?

**Comment by slorber** [comment]
Yes, you can swizzle components and override the default React code with your own logic.

Normally all labels can be customized using our i18n support (but you can also swizzle and hardcode your custom labels if you prefer)
